Userland/Libraries/LibWeb/HTML/HTMLMeterElement.cpp

```diff
@@ -22,4 +26,185 @@ void HTMLMeterElement::initialize(JS::Realm& realm)
     set_prototype(&Bindings::ensure_web_prototype<Bindings::HTMLMeterElementPrototype>(realm, "HTMLMeterElement"));
 }
 
+void HTMLMeterElement::visit_edges(Cell::Visitor& visitor)
+{
+    Base::visit_edges(visitor);
+    visitor.visit(m_meter_value_element);
+}
+
+// https://html.spec.whatwg.org/multipage/form-elements.html#concept-meter-actual
+double HTMLMeterElement::value() const
+{
+    auto maybe_value_string = get_attribute(HTML::AttributeNames::value);
+    if (!maybe_value_string.has_value())
+        return 0;
+    auto maybe_value = parse_floating_point_number(maybe_value_string.value());
+    if (!maybe_value.has_value())
+        return 0;
```

Comment:
The spec describes using 0 as the "candidate actual value", which is then clamped between min and max. So we can't just return 0 here.